WEB_DEPLOYMENT.md

```
@@ -0,0 +1,211 @@
+
+# Terminal 2: Start the frontend dev server
+npm run dev:web
+```
+
+The dev server runs on `http://localhost:5173` with API proxy to the web server.
+
+## Architecture
+
+The web version replaces the Electron main process with a Node.js web server:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)